cms/static/js/views/pages/container_subviews.js

```diff
@@ -50,28 +50,21 @@ define(["jquery", "underscore", "gettext", "js/views/baseview", "js/views/utils/
         });
 
         /**
-         * A controller for updating the "View Live" and "Preview" buttons.
+         * A controller for updating the "View Live" button.
          */
         var PreviewActionController = ContainerStateListenerView.extend({
```

Comment:
Maybe rename this to ViewLiveButtonController now that it has nothing to do with preview.